hls-plugin-api/src/Ide/Types.hs

```diff
@@ -764,6 +770,10 @@ instance PluginRequestMethod Method_TextDocumentDocumentSymbol where
             si = SymbolInformation name' (ds ^. L.kind) Nothing parent (ds ^. L.deprecated) loc
         in [si] <> children'
 
+-- TODO(@linj) is this correct?
```

Comment:
I think this is reasonable. Really `combineResponses` should have a way to return an error. There are a bunch of methods where it really only makes sense if we have one handler.

We _could_ try to combine responses: we would combine the signatures, and so long as only one of them indicated an active signature it would be okay. But that's a bit sketchy and I doubt we'll have several anyway!